<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormClass</class>
 <widget class="QMainWindow" name="MainFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimplePacketEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background:white;
}

QTabWidget::pane
{
background:white;
border:1px solid gray;
}

QTabWidget#tabwMain::pane
{
background:white;
border:none;
border-top:1px solid gray;
border-bottom:1px solid gray;
}

QGroupBox
{
border:1px solid gray;
margin-top:5px;
}

QGroupBox::title
{
color:blue;
top:-7px;
left:5px;
}

QCheckBox:checked
{
color:green;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0" rowspan="4" colspan="3">
     <widget class="QTreeWidget" name="treeWidgetPacketLogList">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBoxLogRecvPacket">
      <property name="text">
       <string>Log Recv</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="checkBoxLogSendPacket">
      <property name="text">
       <string>Log Send</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButtonSendPacket">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditSendPacketText"/>
    </item>
    <item row="2" column="3" rowspan="2">
     <widget class="QTreeWidget" name="treeWidgetModifyHeaderList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainForm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxLogSendPacket</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainFormClass</receiver>
   <slot>CheckBox_Toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLogRecvPacket</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainFormClass</receiver>
   <slot>CheckBox_Toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSendPacket</sender>
   <signal>clicked()</signal>
   <receiver>MainFormClass</receiver>
   <slot>PushButton_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CheckBox_Toggled(bool)</slot>
  <slot>PushButton_Clicked()</slot>
 </slots>
</ui>
